### ISSUE TITLE
Push Responses

### ISSUE THREAD

**Issue by poms32** [davibennun/laravel-push-notification#125]
When I do a push send using a liste of devices using APNS, the repsonse is nul 

array:1 [
  0 => Message {#290
    #id: null
    #code: 0
  }
]

Don't know I have to answer from server

**Comment by manik005** [comment]
hi @poms32 , managed to sort this issue ?

**Comment by poms32** [comment]
Nope, but I didn't try a lot more... sorry, I'm still using AWS SNS

**Comment by manik005** [comment]
ok, thank you. `m trying my luck with it.

**Comment by manik005** [comment]
looks like i is an issue with the main library itself. [https://github.com/Ph3nol/NotificationPusher/issues/104](url) . I tried with the core library without this wrapper still got the same response. as of now I will use `$collection->getFeedback()` , which will return device tokens to which notifications can't/wasn't sent.
